libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/README.md

```diff
@@ -0,0 +1,75 @@
+# AnyLeaf Mercury H7 Flight Controller
+
+The Mercury H7 is a flight controller produced by [AnyLeaf](http://www.anyleaf.org/).
+
+## Features
+    Processor
+        STM32H743 32-bit processor
+    Sensors
+        ICM42688 Acc/Gyro with dedicated 32.768kHz crystal
+        DPS310 barometer
+    Power
+        2S - 6S Lipo input voltage with voltage monitoring
+        9V, 3A supply for powering video transmitters
+        5V, 2A supply for powering servos and electronics
+        3.3V, 1A supply for powering electronics
+    Interfaces
+        8x Bidirectional-DSHOT, or PWM-capable motor outputs
+        1x CAN-FD port for external peripherals
+        1x DJI-format Vtx connector
+        5x UARTs/serial for external peripherals
+        1x I2C for external peripherals
+        USB-C port
+        All UARTS support hardware inversion. SBUS, SmartPort, and other inverted protocols work on any UART without “uninvert hack”
+        Onboard ExpressLRS radio tranceiver
+
+## Pinout
+
+Pins and connector values are labeled on the flight controller PCB.
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (External pad, or DJI HDL)
+ - SERIAL2 -> UART2 (OSD)
+ - SERIAL3 -> UART3 (External pad, or ESC telemetry)
+ - SERIAL4 -> USART4 (External pad)
+ - SERIAL5 -> UART7 (Onboard ELRS control receiver)
+ - SERIAL6 -> UART8 (External pad)
+
+## Can FD port
+
+This flight controller includes a 4-pin DroneCAN standard CAN port. It's capable of 64-byte frames,
+and up to 5Mbps data rates. It's useful for connecting GPS devices, compasses, power monitoring, sensors, motors, servos, and other CAN peripherals.
+
+## RC Input
+
+This flight controller includes a 2.4Ghz ExpressLRS transceiver, capable of receiving control input, and transmitting or receiving MavLink telemetry.
+   
+## OSD Support
+
+This flight controller has an MSP-DisplayPort output on a 6-pin DJI-compatible JST SH port.
+
+## Motor Output
+
+8 Motor outputs are available, mapped to timer peripherals. Each is capable of bidirectional DSHOT,
+or PWM output.
+
+
+## Compass
+
+This flight controller does not have a built-in compass, but you can attach an external compass using the CAN connector, I2C, or Uart.
```

Comment:
AP does not support serial compasses only I2C and DroneCAN

Reply:
line not changed

Reply:
Oop! Done.